parser/lexer/state.go

```diff
@@ -128,6 +135,43 @@ loop:
 	return root
 }
 
+// Process what comes after the env keyword, which must be any
+// set of characters or a string literal, enclosed in square brackets
+func literalIdentifier(l *lexer) stateFn {
```

Comment:
Let's not introduce new syntax. We can just require to use strings:
```
env['foo bar']
```

Reply:
Fair enough - it's not worthwhile to have new syntax for the sake of a two character savings.